Marlin/src/lcd/e3v2/common/encoder.cpp

```diff
@@ -169,13 +169,15 @@ EncoderState encoderReceiveAnalyze() {
 #if PIN_EXISTS(LCD_LED)
 
   // Take the low 24 valid bits  24Bit: G7 G6 G5 G4 G3 G2 G1 G0 R7 R6 R5 R4 R3 R2 R1 R0 B7 B6 B5 B4 B3 B2 B1 B0
-  uint16_t LED_DataArray[LED_NUM];
+  uint32_t LED_DataArray[LED_NUM];
 
   // LED light operation
   void LED_Action() {
     LED_Control(RGB_SCALE_WARM_WHITE, 0x0F);
+    //LED_GraduallyControl(RGB_SCALE_COOL_WHITE, 0x0F, 1000);
     delay(30);
     LED_Control(RGB_SCALE_WARM_WHITE, 0x00);
+    //LED_GraduallyControl(RGB_SCALE_COOL_WHITE, 0x00, 1000);
```

Comment:
I think there is a reason LED_GraduallyControl wasn't used anywhere. It has the potential to loop for a very long time, which probably is not desired.

Even if it is ok, the `1000` argument you have is going to make it wait a full second between each adjustment.

Have you tried to actually enable and use this? Reading the code, I wonder if the more appropriate action is to just delete the `LED_GraduallyControl` function completely.

Reply:
i think graduallycontrol is basically just that, to loop through colors or brightness. the function exists and isn't used. so maybe the original code when first place can say something about it. otherwise it is what it is.